app-typescript/components/TreeSelect/TreeSelect.tsx

```diff
@@ -42,6 +42,8 @@ interface IPropsBase<T> extends IInputWrapper {
     singleLevelSearch?: boolean;
     placeholder?: string;
     searchPlaceholder?: string;
+    noResultsFoundMessage?: string;
+    openedDropdown?: boolean;
```

Comment:
update naming to `dropdownInitiallyOpen` to clarify that the prop applies only to first render.